ModelFactoryCollection/ModelFactoryCollection.php

```diff
@@ -71,10 +71,17 @@ public function createModels($objects)
         // If that fails, try to find model factory for each item individually.
         $models = [];
         foreach ($objects as $key => $object) {
+            if (!is_object($object)) {
```

Comment:
I totally agree. How about just tweaking the message to tell more about the type of the variable passed to `createModels`?